dfgm_sim/dfgm_sim.py

```diff
@@ -1,123 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Is this file no longer useful? If it is, should we keep it (perhaps somewhere else?)